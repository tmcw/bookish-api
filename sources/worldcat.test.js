var WorldCat = require("./worldcat");

test("WorldCat", async () => {
  const wc = new WorldCat();
  expect(wc).toBeTruthy();
  expect(await wc.ISBN("0812993543")).toMatchSnapshot();
});
