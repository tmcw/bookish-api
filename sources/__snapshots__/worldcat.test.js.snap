// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorldCat 1`] = `
Object {
  "isbn10": Array [
    "0812993543",
  ],
  "isbn13": Array [
    "9780812993547",
  ],
  "oclc": undefined,
}
`;
